apidoc/Titanium/UI/TabGroup.yml

```diff
@@ -322,6 +305,21 @@ properties:
     type: String
     platforms: [iphone, ipad]
 
+  - name: labelVisibilityMode
+    summary: The label visibility mode determines whether to show or hide labels in the navigation items.
+    description: |
+        Setting the label visibility mode to [LABEL_VISIBILITY_SELECTED](Titanium.UI.Android.LABEL_VISIBILITY_SELECTED) sets the label to only show when
+        selected, setting it to [LABEL_VISIBILITY_LABELED](Titanium.UI.Android.LABEL_VISIBILITY_LABELED) sets the label to always show, and
+        [LABEL_VISIBILITY_UNLABELED](Titanium.UI.Android.LABEL_VISIBILITY_UNLABELED) sets the label to never show.
+
+        Setting the label visibility mode to [LABEL_VISIBILITY_AUTO](Titanium.UI.Android.LABEL_VISIBILITY_AUTO) sets the label to behave as "labeled"
+        when there are 3 items or less, or "selected" when there are 4 items or more.
+
+        NOTE: This property only affects TabGroups with the [TABS_STYLE_BOTTOM_NAVIGATION](Titanium.UI.Android.TABS_STYLE_BOTTOM_NAVIGATION).
+    since: "9.0.0"
+    constants: Titanium.UI.Android.LABEL_VISIBILITY_*
```

Comment:
Need to document these constants - our doc validation is picking this up as an invalid reference because these constants aren't documented.